### DISCUSSION TITLE
Audio control with multiple displays?

### DISCUSSION THREAD

**Discussion by jfmusic** [MonitorControl/MonitorControl#1265] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Discussion

I have 2 LG 35WN65C-B Monitors.
The app sees both.

I can adjust individual monitors from the pull-down.
When I use the keyboard to adjust the overall volume, 
It only affects the one that is set to Main Monitor in the OS.

IF I drop it to one slider for all it doesn’t affect the Apple Cinema Display.

Thanks

**Comment by waydabber** [comment]
Are you using multiple audio outputs at the same time? Most users probably use one of their external displays for audio output. If you are using one of your displays, you should simply disable volume control for the other one (Preferences/Displays/Show advanced settings). You can also override the audio device name that is associated to the display. You can also try changing `Screen to control` to `Change volume for all screens` for keyboard controls.

**Reply by jfmusic** [reply]
I have two LG and 1 Apple cinema, Your app controls the LG volume , ganged as one but won't control the Apple.    So I can
 control the Apple cinema sometimes from the OS , and the LG's from yours but not all three at the same time in any way. So frustrating.

**Reply by waydabber** [reply]
Yes, I understand the frustration. The app only controls the display's built in amplifier via DDC while the Cinema Display uses an Apple proprietary protocol and system level audio control.

**Comment by jfmusic** [comment]
Thanks for the info...
